src/programs/someone-tag.ts

```diff
@@ -8,90 +16,64 @@ const QUESTION_SHEET_ID: string =
   "1eve4McRxECmH4dLWLJvHLr9fErBWcCGiH94ihBNzK_s";
 
 @Command({
-  event: DiscordEvent.MESSAGE,
-  trigger: "@someone",
-  allowedRoles: ["Seek Discomfort"],
-  channelNames: ["chat", "chat-too", "4th-chat", "chat-v"],
-  description: "This handler is for the someone command.",
+  event: DiscordEvent.SLASH_COMMAND,
+  root: "someone",
+  description: "Ping a random memeber.",
+  options: [
+    {
+      type: ApplicationCommandOptionType.Boolean,
+      name: "Online",
+      description: "Should the person be Online.",
+      required: true,
+    },
+  ],
 })
-class SomeoneTag implements CommandHandler<DiscordEvent.MESSAGE> {
-  async handle(message: Message): Promise<void> {
-    const allow = await isAllowed(message.author);
+class SomeoneTag implements CommandHandler<DiscordEvent.SLASH_COMMAND> {
+  async handle(interaction: ChatInputCommandInteraction): Promise<void> {
+    const allow = await isAllowed(interaction.user);
 
     if (!allow) {
-      await Tools.handleUserError(
-        message,
-        "You have already used this command today!"
-      );
+      interaction.reply("You have already used this command today!");
```

Comment:
- `await` async functions
- Error responses should be ephermal

```suggestion
      await interaction.reply({ content: "You have already used this command today!", ephermal: true });
```

Reply:
not sure if VSC is right, but it gives me an error saying that "ephermal" doesn't exist on type "InteractionReplyOptions" .